project_resource_calendar/models/calendar_event.py

```diff
@@ -50,7 +50,17 @@ class CalendarEvent(models.Model):
         default=False,
     )
     start_datetime = fields.Datetime(
-        'Start DateTime',
+        'Start Date',
```

Comment:
No need to change the name of field, keep the default name for the moment, because it is a DateTime (with hour)

Reply:
@rimbendhaou Done